test/importers/html2jcr/fixtures/richtext-withimage.md

```diff
@@ -0,0 +1,24 @@
++-------------------------------------------------------------------------+
+| richtext2                                                               |
```

Comment:
I'd rename every occurence of `richtext2` to `richtext-withimage` for easier readability

Reply:
Fixed aka renamed